src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/dummy/DummyPlatform.java

```diff
@@ -75,11 +71,7 @@ public String getName() {
      */
     @Override
     public String getID() {
-        String appendix = "";
-        if (name != null) {
-            appendix = name.toLowerCase();
-        }
-        return ID+appendix;
+        return (ID + " " + name).toLowerCase().replaceAll("[^a-z0-9]", "_");
```

Comment:
Or even furthur move it to the interface, so the ID gets computed everytime with this regax